src/axom/core/ArrayBase.hpp

```diff
@@ -303,16 +314,41 @@ class ArrayBase
   void swap(ArrayBase& other)
   {
     std::swap(m_dims, other.m_dims);
+    std::swap(m_spacing, other.m_spacing);
     std::swap(m_strides, other.m_strides);
   }
 
+  /// \brief Set the shape
+  AXOM_HOST_DEVICE void set_shape(const StackArray<IndexType, DIM>& shape_)
```

Comment:
Minor: I believe functions should be in camelCase?

Reply:
You're right.  That would be consistent with the existing function names in the class.